src/generators/createGenerator.js

```diff
@@ -115,6 +140,9 @@ What folder do you want to use as your public web assets directory?
   }
 
   async install () {
+    if (this.options.skip_install) {
+      return
+    }
```

Comment:
We should only ever skip-install if the user replies No to the prompt
If the --yes flag is present, we don't prompt at all, so just install ...

Reply:
fixed